src/game/Spell.cpp

```diff
@@ -2982,9 +2982,12 @@ void Spell::cast(bool skipCheck)
     // CAST SPELL
     SendSpellCooldown();
 
-    TakePower();
-    TakeReagents();                                         // we must remove reagents before HandleEffects to allow place crafted item in same slot
-    TakeAmmo();
+    if (!m_IsTriggeredSpell)
```

Comment:
And now gone is that repo and those commits. Now why would you do that. Doesn't matter, I fetched it locally. Unless you're saying that none of that work was worth anything. Should I go ahead and revert the other commit I pulled from that repo?